src/modules/commander/state_machine_helper.h

```diff
@@ -68,6 +68,32 @@ enum class link_loss_actions_t {
 	LOCKDOWN = 6,		// Kill the motors, same result as kill switch
 };
 
+enum class offboard_loss_actions_t {
+	DISABLED = -1,
+	AUTO_LAND = 0,		// Land mode
+	AUTO_LOITER = 1,	// Hold mode
+	AUTO_RTL = 2,		// Return mode
+	TERMINATE = 3,		// Turn off all controllers and set PWM outputs to failsafe value
+	LOCKDOWN = 4,		// Kill the motors, same result as kill switch
```

Comment:
So I guess there really is a difference between `TERMINATE` and `LOCKDOWN`! I wasn't aware of it :astonished:.

It's probably worth documenting it also in the param docs.

Reply:
Ahah I was not aware of it either. In fact I was really just interested in the DISABLED option. I added these two for consistency with link loss actions. I am still unsure what is the difference between these two. Maybe we open another issue, asking for clarification / pruning in these options?

Reply:
Normally , Terminate sets the failsafe values and lockown sets the disarmed values.

Reply:
Thanks @bresch! Does TERMINATE really turn off all controller? And does LOCKDOWN not?

Reply:
Comments updated